packages/stubGenerator/languages/javascript/src/index.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/no-cycle */
+import { match } from 'ts-pattern';
+
+import {
+  parseLoopCommand,
+  parseLooplineCommand,
+  parseReadCommand,
+  parseWriteCommand,
+} from './commandParsers/';
+
+export const parseLine = (line: string) => {
+  const [command, ...rest] = line.trim().split(' ');
+  return match(command)
+    .with('read', () => parseReadCommand(rest))
+    .with('loopline', () => parseLooplineCommand(rest))
+    .with('loop', () => parseLoopCommand(rest))
+    .with('write', () => parseWriteCommand(rest))
+    .otherwise(() => '');
+};
+
+export const generateJavascriptStub = (input: string) => {
+  const utils = `const toInt = x => Number.parseInt(x, 10);
+const toFloat = x => Number.parseFloat(x, 10);
+const range = n => [...Array(n)].map((_, i) => i);\n\n`;
```

Comment:
I'm not sure that's a good idea.
maybe we can expose this functionality in a global.
`CodeIt.range` and we can get rid of `toInt/toFloat` by `readInt`, `readFloat` abstractions. 

I'm not sure, we should consider other languages also.

Reply:
I mean a built-in helper functions like `readline` but for reading numbers.
In that case, we can eliminate these lines of code in every template.

Reply:
great idea